source/includes/usage-examples/code-snippets/InsertMany.java

```diff
@@ -0,0 +1,39 @@
+// ignored first line
+package usage.examples;
```

Comment:
At some point we should fix up the directory structure here such that we can easily compile this code.

Reply:
Sure, I can change them now if you let me know what the package name should be for automated testing.

Reply:
The package name can be anything.  It's more that the source code has to be in a structure that is consumable by Maven or Gradle (i.e. for this file in `src/main/java/usage/examples`)

Reply:
Oh I see. We'll probably need a script to copy these over to where they can be built, but I'll think about a different parent directory name to implement in a future PR.